docs: Clarify multi-file behavior (#15254)
influxdata/telegraf: docs/TOML.md

users.

Users should be aware that when Telegraf reads a user's config, if multiple

> files or directories are read in, each file is read one at a time and all
> plugins are combined into a single running Telegraf process.

Reviewer: Hmmm it's not only the plugins that are combined, but _all_ config options... How about
```suggestion
files or directories are read in, each file at a time and all
settings are combined as if it were one big file.
```
